engine-rest/engine-rest-openapi/src/main/templates/paths/variable-instance/{id}/get.ftl

```diff
@@ -0,0 +1,81 @@
+<#-- Generated From File: camunda-docs-manual/public/reference/rest/variable-instance/get/index.html -->
+{
+  <@lib.endpointInfo
+      id = "getVariableInstance"
+      tag = "Variable Instance"
+      summary = "Get Variable Instance"
+      desc = "Retrieves a variable by id."
+  />
+
+  "parameters" : [
+
+      <@lib.parameter
+          name = "id"
+          location = "path"
+          type = "string"
+          required = true
+          desc = "The id of the variable instance."
+      />
+
+    <@lib.parameter
+          name = "deserializeValue"
+          location = "query"
+          type = "boolean"
+          desc = "Determines whether serializable variable values (typically variables that store custom Java objects)
```

Comment:
you can reuse `deserialize-values-parameter`

Reply:
I think unfortunately not. The parameter in deserialize-value-parameter.ftl has an additional "s" . Thus, I do not see a way to avoid duplicating it. I checkt and this is true for the code as well, so it is not just a typo they actually have different names.